R/utils.R

```diff
@@ -13,6 +13,48 @@ setup_project <- function() {
   source(path)
 }
 
-use_r_packages <- function(path = "deduplicate/load-and-attach-r-packages.R") {
+use_r_packages <- function(path = packages_path()) {
   source(path)
+  conflicted::conflict_prefer("filter", "dplyr")
+  conflicted::conflict_prefer("lag", "dplyr")
+
+  invisible(path)
+}
+
+packages_path <- function() {
+  "deduplicate/load-and-attach-r-packages.R"
+}
+
+create_install_calls_for_dockerfile <- function(path = packages_path()) {
```

Comment:
seems like this functions purpose is duplicated in the `update_dockerfile_packages` function?